src/component/useFileUpload.js

```diff
@@ -84,15 +99,27 @@ export function useFileUpload({
   )
 
   const validateFileInput = useCallback(
-    file => {
+    files => {
+      let filteredFiles = []
       const validators = []
       validators.push(customValidation)
       if (acceptableextensions.length) {
         validators.push(fileExtensionValidation)
       }
-      validators.map(validate => !file[ERROR] && validate(file))
+      files?.map(file => {
+        validators.map(validate => validate(file))
```

Comment:
This will return the array from the last validator. We need change the logic such a way that if any validator returns error then skip the next validator.

Reply:
Yeah, Will look into. Preview it was not stopping even it has any error with any of the validator